### PR TITLE
Adding TLS certification section on private registry

### PR DESCRIPTION
**Pull request by knabben**
Documenting the option for authenticating the registry with TLS certifications

Ref https://github.com/kubernetes-sigs/kind/issues/1956

### CODE REVIEW

File: site/content/docs/user/private-registries.md [hunk 1]
```diff
@@ -87,3 +87,24 @@ See Google's [upstream docs][keyFileAuthentication] on key file authentication f
...
+resides on your host folder, it is possible to mount and use them into the `containerd` plugin
+patching the default configuration, like in the example:
+
+```
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    extraMounts:
+      - containerPath: /etc/docker/certs.d/registry.dev.databricks.com
```
Comment (tao12345666333) [inline site/content/docs/user/private-registries.md:103]: Thanks! Can you change the domain to example.com?

Comment (knabben) [reply site/content/docs/user/private-registries.md]: @tao12345666333 PTAL

File: site/content/docs/user/private-registries.md [hunk 2]
```diff
@@ -87,3 +87,24 @@ See Google's [upstream docs][keyFileAuthentication] on key file authentication f
 [loading an image]: /docs/user/quick-start/#loading-an-image-into-your-cluster
 [using a private registry]: https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry
 [GCR]: https://cloud.google.com/container-registry/
+
+#### Use a Certificate
+
+If you have a registry authenticated with certificates, and both certificates and keys
+resides on your host folder, it is possible to mount and use them into the `containerd` plugin
```
Comment (BenTheElder) [inline site/content/docs/user/private-registries.md:94]: s/resides/resides

File: site/content/docs/user/private-registries.md [hunk 3]
```diff
@@ -87,3 +87,24 @@ See Google's [upstream docs][keyFileAuthentication] on key file authentication f
...
+patching the default configuration, like in the example:
+
+```
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+  - role: control-plane
+    extraMounts:
```
Comment (BenTheElder) [inline site/content/docs/user/private-registries.md:102]: Can we comnent the config to explain what this is doing?

File: site/content/docs/user/private-registries.md [hunk 4]
```diff
@@ -87,3 +87,24 @@ See Google's [upstream docs][keyFileAuthentication] on key file authentication f
...
 [using a private registry]: https://kubernetes.io/docs/concepts/containers/images/#using-a-private-registry
 [GCR]: https://cloud.google.com/container-registry/
+
+#### Use a Certificate
+
+If you have a registry authenticated with certificates, and both certificates and keys
+resides on your host folder, it is possible to mount and use them into the `containerd` plugin
+patching the default configuration, like in the example:
+
+```
```
Comment (BenTheElder) [inline site/content/docs/user/private-registries.md:97]: Let's make this copyable, see: https://kind.sigs.k8s.io/docs/contributing/development/#shortcodes

**Review by BenTheElder — COMMENTED** [review]
Thanks for the PR!

File: site/content/docs/user/private-registries.md (hunk 3 above)
Comment (knabben) [reply site/content/docs/user/private-registries.md:104]: sure, lmk if this comment works.

**Comment by BenTheElder** [comment]
https://deploy-preview-1962--k8s-kind.netlify.app/

**Review by tao12345666333 — COMMENTED** [review]
/lgtm

:+1:

**Comment by BenTheElder** [comment]
/approve
thank you

**Pull request merged**